rel-eng/custom/custom.py

```diff
@@ -55,7 +55,7 @@ def fetch(self):
                 self.builder.project_name),
             self.spec_file)
         for s in os.listdir(self.builder.start_dir):
-            if os.path.exists(os.path.join(self.builder.start_dir, s)):
+            if os.path.exists(os.path.join(self.builder.start_dir, s)) and os.path.isfile(s):
```

Comment:
In this case ```s``` is still relative so you need to join, but it's better to just replace the ```exists()``` with ```isfile()``` since ```isfile()``` implies ```exists()```.